playbooks/robusta_playbooks/pod_investigator_enricher.py

```diff
@@ -162,9 +162,17 @@ def report_pod_issue(
         event.add_enrichment(blocks)
 
     if reason:
+        # Update findings' descriptions.
         if message is None:
             message = "unknown"
-        event.add_enrichment([MarkdownBlock(f"\n\n{reason}: {message}")])
+        # The code below is essentially a specialized version of ExcutionBaseEvent.override_finding_attributes
```

Comment:
I would put this on the parent class, and a new method `extend_description` or `add_to_description`
`def add_to_description(addition: str)`

Reply:
done